docs/validators/setup/priority_mempool.md

```diff
@@ -0,0 +1,50 @@
+<!--
+order: 5
+-->
+
+# Prioritized Mempool
```

Comment:
We can keep it general and add a section for the `## FIFO Mempool` and another one for the `## Prioritized Mempool`

```suggestion
# Mempool
```